pom.xml

```diff
@@ -91,6 +91,12 @@
 			<artifactId>logback-classic</artifactId>
 			<version>1.2.3</version>
 		</dependency>
+		
+		<dependency>
+ 			 <groupId>jline</groupId>
+  			 <artifactId>jline</artifactId>
+  			 <version>2.14.2</version>
```

Comment:
Isn't 3.x the most recent version? From https://github.com/jline/jline3:

>JLine 3.x is an evolution of JLine 2.x.

Reply:
It seems, that the ConsoleReader is only in the version Jline2.

Reply:
There are new alternatives: https://github.com/jline/jline3/wiki/Using-line-readers